var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -13,13 +13,58 @@ class Crtm(CMakePackage):
     scattering, and a solver for a radiative transfer."""
 
     homepage = "https://www.jcsda.org/jcsda-project-community-radiative-transfer-model"
-    url = "https://github.com/NOAA-EMC/EMC_crtm/archive/refs/tags/v2.3.0.tar.gz"
+    git = "https://github.com/JCSDA/crtm.git"
+    url = "https://github.com/JCSDA/crtm/archive/refs/tags/v2.3.0.tar.gz"
 
     maintainers = [
+        "BenjaminTJohnson",
         "t-brown",
         "edwardhartnett",
         "AlexanderRichert-NOAA",
         "Hang-Lei-NOAA",
+        "climbfuji",
     ]
 
-    version("2.3.0", sha256="3e2c87ae5498c33dd98f9ede5c39e33ee7f298c7317b12adeb552e3a572700ce")
+    variant(
+        "fix",
+        default=False,
+        description='Download CRTM coeffecient or "fix" files (several GBs).',
+    )
+    variant(
+        "build_type",
+        default="RelWithDebInfo",
+        description="CMake build type",
+        values=("Debug", "Release", "RelWithDebInfo", "MinSizeRel"),
+    )
+
+    depends_on("cmake@3.15:")
+    depends_on("git-lfs")
+    depends_on("netcdf-fortran", when="@2.4.0:")
+    depends_on("netcdf-fortran", when="@v2.3-jedi.4")
+    depends_on("netcdf-fortran", when="@v2.4-jedi")
+    depends_on("netcdf-fortran", when="@v2.4-jedi.1")
+    depends_on("netcdf-fortran", when="@v2.4-jedi.2")
+
+    depends_on("crtm-fix@2.3.0_emc", when="@2.3.0 +fix")
+    depends_on("crtm-fix@2.4.0_emc", when="@2.4.0 +fix")
+
+    depends_on("ecbuild", type=("build"), when="@v2.3-jedi.4")
+    depends_on("ecbuild", type=("build"), when="@v2.4-jedi")
+    depends_on("ecbuild", type=("build"), when="@v2.4-jedi.1")
+    depends_on("ecbuild", type=("build"), when="@v2.4-jedi.2")
+
+    # ecbuild release v2.4.0 is broken
+    # add ecbuild dependency for next release with fix
+    # depends_on("ecbuild", when="@2.4.0:", type=("build"))
+
+    # REL-2.4.0_emc (v2.4.0 ecbuild does not work)
+    version("2.4.0", commit="5ddd0d6")
+    # Uses the tip of REL-2.3.0_emc branch
+    version("2.3.0", commit="99760e6")
+    # JEDI applications so far use these versions
+    # Branch release/crtm_jedi
+    version("v2.3-jedi.4", commit="bfede42")
+    # Branch release/crtm_jedi_v2.4.0
+    version("v2.4-jedi", commit="0ee3593")
+    version("v2.4-jedi.1", commit="8222341")
+    version("v2.4-jedi.2", commit="62831cb")
```

Comment:
Are you sure about the use of "v" prefix for some versions and not others?

Reply:
Yes, that's unfortunately the syntax JCSDA has chosen.